packages/api/src/server.ts

```diff
@@ -102,6 +121,11 @@ export class WepublishServer {
       cors: corsOptions
     })
 
+    app.use((err: any, req: Express.Request, res: Express.Response, next: NextFunction) => {
```

Comment:
Not sure if I misunderstand something here but where do you use `req` and `res` in this function? I only see `err` being used

Reply:
because `next` is also used